### ISSUE TITLE
Release v1.5

### ISSUE THREAD

**Issue by adamkaplan** [datatheorem/TrustKit#87]
You've got 4 months of hotness built up and we want to use it!

Doesn't appear to be any breaking changes, I could be wrong about that.

**Comment by nabla-c0d3** [comment]
Agreed =). The last change I want to put in is the expiration date (which is almost done). Hopefully I will release it at the end of the week.

**Comment by adamkaplan** [comment]
Ok, great!

**Comment by adamkaplan** [comment]
@nabla-c0d3 How's it going? Look good to release tomorrow?

Sorry to be a pest. I have a SDK that requires the next release as a dependency via Cocoapods. It's literally impossible to point an SDK transitive dependency at a specific commit, so I have a growing list of apps here with hacky local overrides in their Podfile, like this:

```
  # Override the TrustKit version in Pincushion because DataTheorem has not released
  # 1.5 yet (and XXX certs require 1.5). Once it is released, we can remove this
  # override entirely and rely on the transitive dependency in XXX.
  pod 'TrustKit', :git => 'git@github.com:datatheorem/TrustKit.git', :commit => 'master'
```

**Comment by nabla-c0d3** [comment]
I'm writing the tests now - if it's not done today it will be on Monday.

**Comment by nabla-c0d3** [comment]
Actually it's released now (as 1.4.1).

**Comment by adamkaplan** [comment]
Awesome thank you
